qemu/tests/cfg/qemu_guest_agent.cfg

```
@@ -462,10 +469,15 @@
         - gagent_check_blacklist:
             only Linux
             gagent_check_type = blacklist
-            black_list_spec = BLOCK_RPCS
+            black_list_spec = allow-rpcs
+            black_list_spec_replace = block-rpcs
+            black_list_change_cmd = sed -i 's/${black_list_spec}.*/${black_list_spec_replace}=guest-info"/g' /etc/sysconfig/qemu-ga
             Host_RHEL.m7, Host_RHEL.m8, Host_RHEL.m9.u0, Host_RHEL.m9.u1:
-                black_list_spec = BLACKLIST_RPC
-            black_list_change_cmd = "sed -i 's/${black_list_spec}.*/${black_list_spec}=guest-info/g' /etc/sysconfig/qemu-ga"
+                black_list_change_cmd = "sed -i 's/${black_list_spec_old}.*/${black_list_spec_old}=guest-info/g' /etc/sysconfig/qemu-ga"
+            Host_RHEL.m9.u2, Host_RHEL.m9.u3:
+                black_list_spec_old = BLOCK_RPCS
+                black_list_change_cmd = "sed -i 's/${black_list_spec_old}.*/${black_list_spec_old}=guest-info/g' /etc/sysconfig/qemu-ga"
+                black_list_check_cmd = "grep -nr '${black_list_spec_old}=.*%s' /etc/sysconfig/qemu-ga"
```

Same here, would this met your expectation?
```suggestion
            black_list_spec = allow-rpcs
            black_list_spec_replace = block-rpcs
            Host_RHEL.m7, Host_RHEL.m8, Host_RHEL.m9.u0, Host_RHEL.m9.u1:
                black_list_spec = BLACKLIST_RPC
                black_list_spec_replace = ${black_list_spec}
            Host_RHEL.m9.u2, Host_RHEL.m9.u3:
                black_list_spec = BLOCK_RPCS
                black_list_spec_replace = ${black_list_spec}
            black_list_check_cmd = "grep -nr '${black_list_spec}=.*%s' /etc/sysconfig/qemu-ga"
            black_list_change_cmd = sed -i 's/${black_list_spec}.*/${black_list_spec_replace}=guest-info"/g' /etc/sysconfig/qemu-ga
```

---

1. '*_old' and 'black_list_check_cmd' can be changed as above.
2. 'black_list_change_cmd' can't be merged, because they're not the same param.